sinonjs/sinon
# Revert package-lock.json to v1

Despite https://github.com/npm/cli/issues/1905 being fixed for `npm@7.1.2`, there are still issues with installing `puppeteer` using `npm@7`.

See https://github.com/puppeteer/puppeteer/issues/6586#issuecomment-754038437

Therefore, we're reverting the accidental schema upgrade from 0d1e40bf6d1603791764330947c25e7877f4956c to
the previous version, and removing `markdownlint-cli` once more.

 #### How to verify
1. Check out this branch
2. `npm ci`
3. Observe that Chromium is downloaded
4. Observe that tests pass
